mail-client/bower/metadata.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>gentoo@seichter.de</email>
+		<name>Ralph Seichter</name>
+	</maintainer>
+	<maintainer type="project">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
+	<longdescription>
+		Bower is a curses frontend for the Notmuch email system, written in Mercury.
+	</longdescription>
+	<use>
+		<flag name="tinfo">
```

Comment:
Don't expect users to set USE flags when you can figure out the correct thing to do yourself. If you can't use a proper configure check, use `has_version` to determine flags on `ncurses:0`.